### ISSUE TITLE
With talkback on unable to change profile picture for a learner

### ISSUE THREAD

**Issue by KolliAnitha** [oppia/oppia-android#4369]
**Describe the bug**
With talkback on unable to change profile picture for a learner

**To Reproduce**
1. Login to Oppia as a learner
2. Tap on Navigation menu
3. Click on Profile picture
4. Tap on edit icon - Choose from Gallery
5. Select a picture 

Actual Result:
With talkback on unable to select a picture

https://user-images.githubusercontent.com/101634267/169788930-573dce4f-07c9-4803-9f2c-6201cda11c66.mp4

**Expected behavior**
With talkback on user should be able to select and upload a picture

Environment:
Device name: One plus Nord2 5G
Android version : Android 11
App version: 0.7-alpha-6c08d882d3

**Comment by rt4914** [comment]
I have checked this one my device (Moto G60, Android Version 12) and I was able to successfully  view & update the profile picture.


https://user-images.githubusercontent.com/9396084/231124400-608ecd4b-1193-4d4d-aa9b-7359efc8fb22.mp4

**Comment by rt4914** [comment]
@KolliAnitha Based on this above video I posted, can you check with latest beta app whether we can close this issue?

**Comment by KolliAnitha** [comment]
@rt4914, hey Rajat I verified in the latest beta build, now i am able to update the profile picture when talkback is enabled. We can close the issue.
